client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -447,12 +447,17 @@ const siteSetupFlow: Flow = {
 					if ( providedDependencies?.type === 'redirect' ) {
 						return exitFlow( providedDependencies?.url as string );
 					}
-
 					switch ( providedDependencies?.action ) {
 						case 'verify-email':
 							return navigate( `verifyEmail?${ urlQueryParams.toString() }` );
 						case 'checkout':
 							return exitFlow( providedDependencies?.checkoutUrl as string );
+						case 'customized-action-flow': {
+							const migrateEntireSiteFlow = urlQueryParams.get( 'migrateEntireSiteFlow' );
```

Comment:
I think `migrateEntireSiteFlow` should be more generic, but we can fix it in another PR

Reply:
Makes sense! I'll update this. Thank you, Gabriel!